jwst/fits_generator/create_data.py

```diff
@@ -25,8 +25,11 @@ def get_proposals(base_directory='.'):
     proposal_list = []
     for dirname, subdirlist, filelist in os.walk(base_directory):
         for name in filelist:
-            if name.endswith('.prop'):
-                proposal_list.append((dirname, name))
+            if name.startswith('.'):
+                pass
```

Comment:
Or if you prefer to stick with this format, `continue` should be used here instead of `pass` as you do just want to ignore these files